src/components/form/form.scss

```
@@ -0,0 +1,195 @@
+@use 'styles/utils';
+
+.question-form {
+  @include utils.bg-defaults(cover);
+  background-image: url("assets/form-bg.jpg");
+  padding-top: 148px;
+  position: relative;
+
+  &::after {
+    content: '';
+    position: absolute;
+    inset: 0;
+    background-color: hsla(245, 96%, 17%, 0.3);
+  }
+
+  .container {
+    position: relative;
+    z-index: 2;
+  }
+
+  .section-title-wrapper {
+    margin-bottom: 60px;
+  }
+
+  &__wrapper {
+    padding-bottom: 60px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    gap: 24px;
+  }
+
+  &__content {
+
+    .title {
+      @include utils.text-header-100;
+      margin-bottom: 24px;
+      color: var(--base100);
+    }
+
+    .text {
+      @include utils.text-base-200-lg-100;
+      color: var(--base100);
+      opacity: .5;
+      margin-bottom: 39px;
+    }
+
+    .socials {
+      display: flex;
+      align-items: center;
+      gap: 8px;
+
+      &__img {
+        width: 80px;
+        height: 80px;
+        border-radius: 50%;
+        background-color: hsla(0, 0%, 100%, 0.1);
+        display: flex;
+        align-items: center;
+        justify-content: center;
+
+        img {
+          filter: brightness(10);
+        }
+      }
+    }
+  }
+
+  form {
+    min-width: 753px;
+    background-color: hsla(0, 0%, 100%, 0.2);
+    border-radius: 40px;
+    padding: 70px;
+    display: flex;
+    flex-direction: column;
+    gap: 128px;
+    position: relative;
+    overflow: hidden;
+
+
+    &::after {
+      content: '';
+      position: absolute;
+      inset: 0;
+      z-index: 1;
+      backdrop-filter: blur(10px);
+
+    }
+  }
+
+  &__inputs {
+    position: relative;
+    z-index: 2;
+    display: flex;
+    flex-direction: column;
+    gap: 48px;
+
+  }
+
+  textarea {
+    @include utils.custom-scroll(10px, transparent, white);
+    background-color: transparent;
+    resize: none;
+    border: none;
+    outline: none;
+  }
+
+  input, textarea {
+    @include utils.text-base-200-lg-100;
+    border-bottom: 1px solid hsla(0, 0%, 100%, 0.2);
+    padding-bottom: 14px;
+    color: var(--base100);
+
+    &::placeholder {
+      @include utils.text-header-400;
+      color: var(--base100);
+    }
+  }
+
+  &__footer {
+    position: relative;
+    z-index: 2;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    p {
+      @include utils.text-base-300-lg-100;
+      color: hsl(0, 0%, 100%, 0.5);
+      width: 272px;
```

Не используй фиксированные ширины. Из-за них адаптив неверно работает.

```suggestion
      max-width: 272px;
      width: 100%;
```

---

исправила